packages/react-dom/src/server/ReactDOMServerExternalRuntime.js

```diff
@@ -1,18 +1,164 @@
-// TODO: Add Flow types
+/**
+ * This file is compiled to a standalone browser script by rollup and loaded by Fizz
+ *  clients. Therefore, it should be fast and not have many external dependencies.
+ * @flow
+ */
+
+// Imports are resolved statically by the closure compiler in release bundles
+// and by rollup in jest unit tests
 import {
   clientRenderBoundary,
   completeBoundaryWithStyles,
   completeBoundary,
   completeSegment,
-} from 'react-dom-bindings/src/server/fizz-instruction-set/ReactDOMFizzInstructionSet';
-
-// Intentionally does nothing. Implementation will be added in future PR.
-// eslint-disable-next-line no-unused-vars
-const observer = new MutationObserver(mutations => {
-  // These are only called so I can check what the module output looks like. The
-  // code is unreachable.
-  clientRenderBoundary();
-  completeBoundaryWithStyles();
-  completeBoundary();
-  completeSegment();
-});
+} from '../../../react-dom-bindings/src/server/fizz-instruction-set/ReactDOMFizzInstructionSet';
```

Comment:
Maybe we should just move this file to the `react-dom-bindings` package? I haven't quite internalized what the boundary is yet cc: @gnoff

Reply:
(I know this already landed in the earlier PR, just didn't think about it until now)

Reply:
Ahh my bad - I struggled to find the correct rollup option 😅
Should I land this fix as a separate PR?

Reply:
bindings is good for basically everything except entrypoints. i would move this file

Reply:
> bindings is good for basically everything except entrypoints. i would move this file

Created #25617 for this change. I'm not familiar with some of the design decisions for bundling / directory layout, so would love some feedback on this